cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -92,10 +96,23 @@ public ClusterOperatorConfig(Set<String> namespaces, long reconciliationInterval
      * @return  Cluster Operator configuration instance
      */
     public static ClusterOperatorConfig fromMap(Map<String, String> map) {
+        warningsForRemovedEndVars(map);
         KafkaVersion.Lookup lookup = parseKafkaVersions(map.get(STRIMZI_KAFKA_IMAGES), map.get(STRIMZI_KAFKA_CONNECT_IMAGES), map.get(STRIMZI_KAFKA_CONNECT_S2I_IMAGES), map.get(STRIMZI_KAFKA_MIRROR_MAKER_IMAGES), map.get(STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES));
         return fromMap(map, lookup);
     }
 
+    /**
+     * Logs warnings for removed / deprecated environment variables
+     *
+     * @param map   map from which loading configuration parameters
+     */
+    private static void warningsForRemovedEndVars(Map<String, String> map) {
+        if (map.get(STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE) != null)    {
```

Comment:
just `map.containsKey`? or any specific reason for getting the value and see if it's `null`?

Reply:
It is technically the same, but I changed it to `containsKey`.